contracts/rbn-staking/IncentivizedVotingLockup.sol

```diff
@@ -92,6 +130,15 @@ contract IncentivisedVotingLockup is
     uint128 blk;
   }
 
+  struct Boost {
+    uint256 delegatedBias;
+    int128 delegatedSlope;
+    uint256 receivedBias;
+    int128 receivedSlope;
+    uint128 nextExpiry;
+    uint32 fromBlock;
+  }
```

Comment:
There are ways we can better [pack this struct](https://fravoll.github.io/solidity-patterns/tight_variable_packing.html)

Try this:

```solidity
struct Boost {
    // First slot
    int128 receivedSlope;
    int128 delegatedSlope;

    // Second slot
    uint128 nextExpiry;
    uint128 fromBlock;
   
    // Third slot
    uint128 delegatedBias;
    uint128 receivedBias;
  }
```

Reply:
yeah was planning on making these optimizations in subsequent prs